app/views/distributions/index.html.erb

```diff
@@ -37,19 +37,24 @@
             <%= form_tag(distributions_path, method: :get) do |f| %>
               <div class="row">
                 <% if @items.present? %>
-                  <div class="form-group col-lg-3 col-md-3 col-sm-6 col-xs-12">
+                  <div class="form-group col-lg-2 col-md-2 col-sm-6 col-xs-12">
                     <%= filter_select(label: "Filter by item", scope: :by_item_id, collection: @items, selected: @selected_item) %>
                   </div>
                 <% end %>
                 <% if @partners.present? %>
-                  <div class="form-group col-lg-3 col-md-3 col-sm-6 col-xs-12">
+                  <div class="form-group col-lg-2 col-md-2 col-sm-6 col-xs-12">
                     <%= filter_select(scope: :by_partner, collection: @partners, selected: @selected_partner) %>
                   </div>
                 <% end %>
-                <div class="form-group col-lg-3 col-md-3 col-sm-6 col-xs-12">
+                <% if @storage_locations.present? %>
+                  <div class="form-group col-lg-2 col-md-2 col-sm-6 col-xs-12">
```

Comment:
```suggestion
                  <div class="form-group col-lg-3 col-md-3 col-sm-6 col-xs-12">
```
Could we expand this a bit? We ended up reverting some styling changes that you were working off of. This suggestion would prevent the label from pushing the filter input down:

<img width="1371" alt="Screen Shot 2020-12-25 at 5 43 59 PM" src="https://user-images.githubusercontent.com/11335191/103142841-fefff680-46d8-11eb-883f-3f288279293f.png">